NibView.swift

```diff
@@ -6,7 +6,7 @@
 //  Copyright © 2016 Trafi. All rights reserved.
 //
 
-import NibView
+import UIKit
```

Comment:
We still need it for Carthage installation:
```suggestion
#if canImport(NibView)
import NibView #endif
import UIKit
```